[quant][graph] Add quantized batch_norm2d support to graph mode

supriyar opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #36622 [quant][graph] Graph mode quantization support for sigmoid
* #36552 [quant][graph] Add quantized batch_norm2d_relu to graph mode
* **#36549 [quant][graph] Add quantized batch_norm2d support to graph mode**
* #36548 [quant][graph] Add valueQuantizable function

Summary:

Test Plan:
python test_quantize_script.py

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D21048965](https://our.internmc.facebook.com/intern/diff/D21048965)

supriyar committed:
[quant][graph] Add quantized batch_norm2d support to graph mode
Summary:

Test Plan:
python test_quantize_script.py

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

supriyar committed:
Update on "[quant][graph] Add quantized batch_norm2d support to graph mode"


Summary:

Test Plan:
python test_quantize_script.py

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

supriyar committed:
Update on "[quant][graph] Add quantized batch_norm2d support to graph mode"


Summary:

Test Plan:
python test_quantize_script.py

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

supriyar committed:
Update on "[quant][graph] Add quantized batch_norm2d support to graph mode"


Summary:

Test Plan:
python test_quantize_script.py

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 reviewed (approved):
Looks good, please move the batchnorm change from previous PR to here as well

jerryzh168 commented on test/quantization/test_quantize_script.py:
```diff
@@ -1390,6 +1390,22 @@ def forward(self, x, y):
+        class M(torch.nn.Module):
+            def __init__(self):
+                super(M, self).__init__()
+                self.bn = torch.nn.BatchNorm2d(3).to('cpu', torch.float)
```
I think `cpu` is default?

supriyar committed:
Update on "[quant][graph] Add quantized batch_norm2d support to graph mode"


Summary:

Test Plan:
python test_quantize_script.py

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

supriyar committed:
Update on "[quant][graph] Add quantized batch_norm2d support to graph mode"


Summary:

Test Plan:
python test_quantize_script.py

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

supriyar committed:
Update on "[quant][graph] Add quantized batch_norm2d support to graph mode"


Summary:

Test Plan:
python test_quantize_script.py

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

supriyar committed:
Update on "[quant][graph] Add quantized batch_norm2d support to graph mode"


Summary:

Test Plan:
python test_quantize_script.py

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]
